components/aws/sagemaker/TrainingJob/component.yaml

```diff
@@ -172,12 +172,12 @@ outputs:
   - {
       name: secondary_status,
       type: String,
-      description: "Provides detailed information about the state of the training job. For detailed information on the s",
+      description: "Provides detailed information about the state of the training job.",
     }
   - {
       name: training_job_status,
       type: String,
-      description: "The status of the training job.   Amazon SageMaker provides the following training job statuses:",
+      description: "The status of the training job.",
```

Comment:
adding a comment to remember:
- update the image version to `2.0.0` 
- write changelog
these will be done as part of release process